components/composearea.json

```diff
@@ -0,0 +1,40 @@
+{
+  "composearea": {
+    "comment": "This is applied to the compose area component inside messaging tab.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=29743%3A57187",
+    "divider": {
+      "default": {
+        "#normal": {
+          "background": ["@theme-gradientdivider-normal-0", "@theme-gradientdivider-normal-1"]
```

Comment:
do we need the "background" key on every state?

Reply:
it is never going to be sth else (like text, outline, etc...) cause it is a divider line

Reply:
We don't, will update